tox.ini

```diff
@@ -70,6 +71,11 @@ statistics = True
 disable =
     too-few-public-methods,
     too-many-instance-attributes,
+    unnecessary-pass,
+    unsubscriptable-object,
+    useless-object-inheritance,
+    trailing-comma-tuple,
+    arguments-differ,
```

Comment:
Maybe open an issue to re-enable these checks some day?

Reply:
These checks were never enabled. These checks were added in the new version of pylint that was required to get this working. Feel free to create an issue for this.